Python/other/Moore_Algorithm.py

```diff
@@ -0,0 +1,37 @@
+## This Program Checks the maximum occurence of a number in a array/list
```

Comment:
@ShubhamPatel33 Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

Reply:
Changes Made in the file